boards/arm/ev11l78a/doc/index.rst

```diff
@@ -0,0 +1,89 @@
+.. _ev11l78a:
+
+UPD301C Basic Sink Application Example
+######################################
+
+Overview
+********
+
+The UPD301C Basic Sink Application Example Evaluation Kit (EV11L78A)
+is a low-cost evaluation platform for Microchip's UPD301C Standalone
+Programmable USB Power Delivery (PD) Controller. This RoHS-compliant
+evaluation platform comes in a small form factor and adheres to the
+USB Type-C™ Connector Specification and USB PD 3.0 specification.
+
+.. figure:: img/ev11l78a.jpg
+   :align: center
+   :alt: EV11L78A
+
```

Comment:
indent with 5 spaces + add suggested width of 500px

```suggestion
.. figure:: img/ev11l78a.jpg
     :width: 500px
     :align: center
     :alt: EV11L78A

```

Reply:
Done — though as a note, the indentation needs to be consistent for the figure description or Sphinx will misparse this.